python/lsst/rubintv/production/rubinTv.py

```diff
@@ -422,10 +423,32 @@ def __init__(self, doRaise=False):
         self.client = EfdClient('summit_efd')
         self.log = _LOG.getChild("mountTorqueChannel")
         self.channel = 'auxtel_mount_torques'
+        self.shardsDir = os.path.join(outputRoot, 'shards')
         self.watcher = Watcher(self.dataProduct, self.channel)
         self.fig = plt.figure(figsize=(16, 16))
         self.doRaise = doRaise
 
+    def writeMountErrorShard(self, errors, dataId):
+        """Write a metadata shard for the mount error, including the flag
+        for coloring the cell based on the threshold values.
+        """
+        dayObs = butlerUtils.getDayObs(dataId)
+        seqNum = butlerUtils.getSeqNum(dataId)
+        az_rms, el_rms, _ = errors  # we don't need the rot errors here so assign to _
+        imageError = (az_rms ** 2 + el_rms ** 2) ** .5
+        key = 'Mount motion image degradation'
+        flagKey = '_' + key  # color coding of cells always done by prepending with an underscore
```

Comment:
This key is only added if there's an issue.  Should this be always populated, with a key like "Mount motion quality" `in  {'good', 'warning', 'bad'}`?

Reply:
I think, for now, we don't want to add the good by default. The front end will colour that green, and I think, for now, we just want to leave it grey, but it's certainly potentially to be added in the future, thanks.